### PR TITLE
update(docs): update modern bpf documentation

### PR DESCRIPTION
**Pull request by Andreagit97**
**What type of PR is this?**

 /kind content

**Any specific area of the project related to this PR?**

/area documentation

**What this PR does / why we need it**:

This PR updates the documentation of the modern bpf probe -> no more experimental

**Which issue(s) this PR fixes**:

**Special notes for your reviewer**:

This is for Falco 0.35

### CODE REVIEW

File: content/en/docs/getting-started/running/index.md [hunk 1]
```diff
@@ -267,7 +267,7 @@ docker run --rm -i -t \
 
 #### Least privileged
 
-Right now **is not officialy supported** but the following command should work great:
+The following capabilities should be enough to run Falco:
```
Comment (pmusa) [inline content/en/docs/getting-started/running/index.md:270]: What capabilities are we talking about? Below I can only see a command. Do you mean the following command?

**Comment by pmusa** [comment]
What is the current status of the userspace kernel events? I believe it is currently not maintained. If that is the case, I suggest removing it from the main table, reducing its size, and moving it to the end of the list. What do you think?

File: content/en/docs/getting-started/running/index.md (hunk 1 above)
Comment (Andreagit97) [reply content/en/docs/getting-started/running/index.md]: uhm good point i added them, ty!

**Comment by Andreagit97** [comment]
> What is the current status of the userspace kernel events? I believe it is currently not maintained. If that is the case, I suggest removing it from the main table, reducing its size, and moving it to the end of the list. What do you think?

Completely agree with you! WDYT? @leogr

**Comment by leogr** [comment]
Totally agree 👍

**Comment by FedeDP** [comment]
:+1:  for me; but please let's get this merged ASAP since doc is out of sync :)

**Review by leogr — APPROVED** [review]

**Comment by leogr** [comment]
@Andreagit97, please, if anything is missing, open a follow-up PR.

Thank you! :pray:

**Pull request merged**